includes/class-sm-post-types.php

```diff
@@ -260,7 +260,8 @@ public static function register_post_types() {
 				'wpcom-markdown',
 				'comments',
 				'entry-views',
-				'elementor',
+                'elementor',
+                'revisions',
```

Comment:
Should use tabs :wink:

Reply:
I typically prefer spaces these days too, but since the standard is tabs - then I'm happy to follow the standard. Consistency matters more than tabs or spaces IMO